### ISSUE TITLE
Monkeys can take the Nuclear Authentication Disk into the vents and just stay there forever

### ISSUE THREAD

**Issue by TheVekter** [tgstation/tgstation#86486]
### Issue Summary:

What it says on the tin. This makes it effectively impossible for nukies to get the disk without blowing up part of the station, and the monkey can kinda just... fuck off, so that's not even really that effective. 

is almost assuredly not a bug, but I'm also not convinced it's intentional and it's definitely not good for the game.

Probably sane to just make it so the disk falls out of your hand/pockets if you try to enter the vents.

### Reproduction Steps:

1) Become a monkey (probably through being a Changeling).
2) Get dat fukken disk.
3) Find a vent.
4) Just fuck around all shift and cuck the nukies.

**Comment by deathrobotpunch** [comment]
ok but this would be a ghostdrone nerf inadvertently

**Comment by Xkallubar** [comment]
> ### Reproduction Steps:
> 1. Become a monkey (probably through being a Changeling).
> 2. Get dat fukken disk.
> 3. Find a vent.
> 4. Just fuck around all shift and cuck the nukies.

Monkeys can't carry items into vents. You have to get it implanted into your chest via surgery.

**Comment by deathrobotpunch** [comment]
simple. make it so you can’t implant the nuke disk in your chest

**Comment by TheVekter** [comment]
> > ### Reproduction Steps:
> > 
> > 1. Become a monkey (probably through being a Changeling).
> > 2. Get dat fukken disk.
> > 3. Find a vent.
> > 4. Just fuck around all shift and cuck the nukies.
> 
> Monkeys can't carry items into vents. You have to get it implanted into your chest via surgery. (I was the monkey ling)

Fixed.

**Comment by CCC23** [comment]
I didnt get dat disk fix

**Comment by Likteer** [comment]
They can still bomb the pipes without having to touch any atmos stuff, right?

**Comment by Sadboysuss** [comment]
I don't think this is an exploit. In fact, I don't think this is an issue at all. A pr that fixes this "issue" would be marked as balance.

**Comment by TheVekter** [comment]
> They can still bomb the pipes without having to touch any atmos stuff, right?

Yeah, but you can't effectively bomb the whole pipenet without destroying the station to begin with. The monkey player would have to fuck up big time to get caught by it.

> I don't think this is an exploit. In fact, I don't think this is an issue at all. A pr that fixes this "issue" would be marked as balance.

It's an oversight at the very least; we have proofing for every other instance of the disk being unobtainable by ops, this isn't any different than that. Just a more creative way of doing it. The intent is clearly "The disk shouldn't be somewhere ops can't realistically reach".

**Comment by Sadboysuss** [comment]
> It's an oversight at the very least; we have proofing for every other instance of the disk being unobtainable by ops and this isn't any different than that. Just a more creative way of doing it. The intent is clearly "The disk shouldn't be somewhere ops can't realistically reach".

- I don't think ops can't realistically reach it, it is hard, yes, but not impossible for sure. Sandbox elements like this are what make ss13 fun and this is a clever tactic to secure the disk born from player creativity, do we want to restrict player creativity?
- You could arguably do the same thing by sending the disk through a really long disposals loop, would that be abuse of game features as well? 
- Lastly, I believe this is a feature request for a balance pr. The issue tracker isn't a place to make feature requests, you can do that here: https://tgstation13.org/phpBB/viewforum.php?f=5

**Comment by TheVekter** [comment]
>     * I don't think ops can't realistically reach it, it is hard, yes, but not impossible for sure. Sandbox elements like this are what make ss13 fun and this is a clever tactic to secure the disk born from player creativity, do we want to restrict player creativity?
> 
>     * You could arguably do the same thing by sending the disk through a really long disposals loop, would that be abuse of game features as well?
> 
>     * Lastly, I believe this is a feature request for a balance pr. The issue tracker isn't a place to make feature requests, you can do that here: https://tgstation13.org/phpBB/viewforum.php?f=9

* There are times when creativity needs to be limited to guarantee a fair play experience (see: changeling hand tele chasm dumping)
* I think it's an unreasonable ask and distinctly different than the disposal example because the disk WILL eventually exit disposals if you break the loop, the monkey can and will just move somewhere else. 
* This is an oversight unless proven as intentional. If a maintainer wants to close this as intentional then that's up to them. If you close it, I will be having words with the head coders about use of member powers.

**Comment by Timberpoes** [comment]
Speaking as a maintainer, many issue reports result in balance PRs. Oversights are the biggest one and they overlap neatly with exploits, which occur where our emergent systems interact in ways we never intended (exploits) or never considered (oversights).

I'm not going to declare this an outright intended feature since I think it feels a little exploity or oversighty in nature. It's mixing different gameplay systems which ordinarily don't interact together but are emergently able to.

In a hypothetical world, I could see myself merging a PR to "fix" this, if this gameplay pattern became a problem.

**Comment by Sadboysuss** [comment]
Clever tactics force the other party to use their brain.
- Unwrenching/exploding pipes will eventually corner the vent crawler
- Pushing a burn mix into distro will kill the vent crawler

Of course, I wouldn't close the issue, and I asked for maintainer input in coder gen.

But if it does become a meta and players abuse it, just change the ventcrawl speed to be slower? That seems like a good solution.

**Comment by TheVekter** [comment]
> Clever tactics force the other party to use their brain.
> 
>     * Unwrenching/exploding pipes will eventually corner the vent crawler
> 
>     * Pushing a burn mix into distro will kill the vent crawler
> 
> 
> Of course, I wouldn't close the issue, and I asked for maintainer input in coder gen.
> 
> But if it does become a meta and players abuse it, just change the ventcrawl speed to be slower? That seems like a good solution.

I don't want to sound like an asshole, but what nuke team is going to get to the station, go loud, and use all 5 of their minutes they have before the shuttle gets there to plasma flood the station?

**Comment by Metekillot** [comment]
> Clever tactics force the other party to use their brain.
> - Unwrenching/exploding pipes will eventually corner the vent crawler
> - Pushing a burn mix into distro will kill the vent crawler

I agree with clever tactics; but I don't agree with burdensome cheese tactics, which this nuke disc thing sounds like.

In either of those scenarios, the expectation on nuke ops players is immense BEFORE they have to triangulate the location of a vent crawling monkey while already in a gunfight with 7-1 odds; they have no way to affect the monkey before they would pursue the tactics of

1) realizing that the pinpointer is following a monkey in the vents (totally unintuitive) and spending their limited explosive resources on the pipes OR stashing their weapons to unwrench them
2) learning or already knowing atmospherics to find or construct a point to flood distro or the scrubbers with a mix to flush out said monkey crawler

**Comment by moocowswag** [comment]
> Now, if you want to code up a nuke ops tool that syndicate bluespace compresses a canister into a man portable device that can instantly hook up to a pipe so nukies can flood the second they get five seconds beside a pipe, I think that's a fair compromise and a cool feature add

saboteur borg

**Comment by Dorsisdwarf** [comment]
FWIW I marked this issue as an exploit rather than an oversight because the game generally tries to avoid making it trivial to put the nuclear authentication disk into these kinds of inaccessible situations. 

(I say inaccessible because pipecrawling is a very fast way to move, is invisible and "walks through walls", and the atmospherics pipenet on most stations is sufficiently interconnected that you'd have to flub pretty hard to get cornered)

**Comment by TheVekter** [comment]
> FWIW I marked this issue as an exploit rather than an oversight because the game generally tries to avoid making it trivial to put the nuclear authentication disk into these kinds of inaccessible situations.
> 
> (I say inaccessible because pipecrawling is a very fast way to move, is invisible and "walks through walls", and the atmospherics pipenet on most stations is sufficiently interconnected that you'd have to flub pretty hard to get cornered)

"Effectively inaccessible" is probably a better term because you know some pedant is going to come behind you and go 🤓

**Comment by TheVekter** [comment]
> doesnt seem too different from the disk inside a pai modsuit tactic, to which counterplay is bombing the modsuit user just like in this scenario it is bombing the pipes to cut the monkeys way. gotta keep in mind in this case crew cant even follow up with the disk bearer since they do not see him either which means that as the crew you re completely relying on nukies Not knowing of this tactic.

That also _kind of_ sucks, but at least you know the disk is on someone that you can actually see and interact with.

You can also just kill the person in the modsuit. It's effectively the same as implanting the disk inside someone, and I'm also cool with that for the same reasons. The problem with a monkey having it in the pipes isn't "it's difficult to access and mobile", it's "nobody is ever going to understand that the disk is in a monkey in the pipes because why the fuck would they ever think of that, plus it's also almost impossible to do anything about".

**Comment by NanoCats** [comment]
> simple. make it so you can’t implant the nuke disk in your chest

but implanting the disk in the clown should be a viable strategy!

**Comment by timothymtorres** [comment]
What was the proposed solution? 
- Disabling the disk from being implanted
- Disabling someone from vent crawling with the disk implanted
- Having the disk randomly spawn if someone vent crawls with it

**Comment by projectkepler-ru** [comment]
The issue I feel with disk inside monkey is precisely that a person in a vent is really goddamn fast
